Doc/library/graphlib.rst

```diff
@@ -99,21 +99,23 @@
 
    .. method:: prepare()
 
-      Mark the graph as finished and check for cycles in the graph. If any cycle
-      is detected, :exc:`CycleError` will be raised, but
-      :meth:`~TopologicalSorter.get_ready` can still be used to obtain as many
-      nodes as possible until cycles block more progress. After a call to this
-      function, the graph cannot be modified, and therefore no more nodes can be
-      added using :meth:`~TopologicalSorter.add`.
+      Mark the graph as finished and check for cycles in the graph.
+
+      After calling this method the graph cannot be modified using
+      the :meth:`~TopologicalSorter.add` method.
+
+      Raises :exc:`CycleError` if any cycles are detected,
+      but :meth:`~TopologicalSorter.get_ready` can still be used to obtain as
+      many nodes as possible until cycles block more progress.
 
    .. method:: is_active()
 
       Returns ``True`` if more progress can be made and ``False`` otherwise.
       Progress can be made if cycles do not block the resolution and either
-      there are still nodes ready that haven't yet been returned by
-      :meth:`TopologicalSorter.get_ready` or the number of nodes marked
-      :meth:`TopologicalSorter.done` is less than the number that have been
-      returned by :meth:`TopologicalSorter.get_ready`.
+      there are still nodes ready to be returned by
+      :meth:`~TopologicalSorter.get_ready` or there are nodes which were
+      returned by :meth:`~TopologicalSorter.get_ready` and which have not yet
+      been marked as :meth:`~TopologicalSorter.done`.
```

Comment:
What is the problem with displaying the full name for these methods?  It makes it clear that we’re talking about methods not functions for example.

Reply:
It's too verbose. The phrase `TopologicalSorter.get_ready()` appears twice in the sentence.
IMO it should be obvious to the users that `get_ready` and `done` are methods, especially since they're clickable links.